### ISSUE TITLE
Unused JS code detection in Safari

### ISSUE THREAD

**Issue by pankajparashar** [captainbrosset/devtools-tips#73]
Safari does have a neat way to show unused JS code in the Sources panel. It basically fades the unexecuted parts of the code. I had covered this in [canidev.tools/find-unused-javascript/safari](https://www.canidev.tools/find-unused-javascript/safari).

![find-unused-javascript-safari_u9jd83](https://user-images.githubusercontent.com/38640616/222813118-56afc508-6e26-483f-9101-d5cc22884b31.gif)

**Comment by captainbrosset** [comment]
Thanks Pankaj. Very useful as always!
From CIDT, I see that you only need to click the `C` icon. So, no need to start coverage instrumentation from any other tool, right?

**Comment by pankajparashar** [comment]
Yes correct. Click the "C" icon > Reload the page to fade unexecuted code. Here is a complete demo:


https://user-images.githubusercontent.com/38640616/223223588-a68faa67-6f1b-4fa9-b4b6-1b3b754f98b2.mp4
